src/gitea/pull_request/pull_request.py

```
@@ -0,0 +1,120 @@
+
+from requests import Response
+
+from gitea.pull_request.base import BasePullRequest
+from gitea.resource.resource import Resource
+from gitea.utils.response import process_response
+
+
+class PullRequest(BasePullRequest, Resource):
+    """Synchronous Pull Request API for Gitea."""
+
+    def _list_pull_requests(  # noqa: PLR0913
```

Ruff (0.14.14)

[warning] Unused `noqa` directive (non-enabled: `PLR0913`)

Remove unused `noqa` directive

(RUF100)